---a/net/minecraft/network/login/server/S00PacketDisconnect.java
+++b/net/minecraft/network/login/server/S00PacketDisconnect.java
@@ -1,43 +1,39 @@
 package net.minecraft.network.login.server;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.login.INetHandlerLoginClient;
 import net.minecraft.util.IChatComponent;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S00PacketDisconnect implements Packet
-{
+
+public class S00PacketDisconnect implements Packet {
+
     private IChatComponent reason;
     private static final String __OBFID = "CL_00001377";
 
-    public S00PacketDisconnect() {}
+    public S00PacketDisconnect() {
+    }
 
-    public S00PacketDisconnect(IChatComponent reasonIn)
-    {
+    public S00PacketDisconnect(IChatComponent reasonIn) {
         this.reason = reasonIn;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.reason = buf.readChatComponent();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeChatComponent(this.reason);
     }
 
-    public void processPacket(INetHandlerLoginClient p_180772_1_)
-    {
+    public void processPacket(INetHandlerLoginClient p_180772_1_) {
         p_180772_1_.handleDisconnect(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public IChatComponent func_149603_c()
-    {
-        return this.reason;
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerLoginClient) handler);
     }
-
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerLoginClient)handler);
-    }
-}
+}
